recipes/qt/6.x.x/conanfile.py

```diff
@@ -681,9 +683,17 @@ def build(self):
     def _cmake_executables_file(self):
         return os.path.join("lib", "cmake", "Qt6Core", "conan_qt_executables_variables.cmake")
 
+    @property
+    def _cmake_entry_point_file(self):
+        return os.path.join("lib", "cmake", "Qt6Core", "conan_qt_entry_point.cmake")
+
     def _cmake_qt6_private_file(self, module):
         return os.path.join("lib", "cmake", "Qt6{0}".format(module), "conan_qt_qt6_{0}private.cmake".format(module.lower()))
 
+    @property
+    def _create_entry_point_targets(self):
+        return not self.options.no_entrypoint and self.settings.os in ["Windows", "iOS"]
```

Comment:
Ok, understood. I removed the option completely. At least now it is possible to set `qt_no_entrypoint` property on targets to disable that entry point library.

Reply:
I just wanted to add that actually libraries don't get linked with entry point library regardless of `no_entrypoint` option since the generator expression has `$<STREQUAL:$<TARGET_PROPERTY:TYPE>,EXECUTABLE>` condition.

The `no_entrypoint` option could be still a problem if there are some build utility programs in the tree that link with Qt and expect entry point library linking.